config/system.php

```diff
@@ -1703,4 +1710,4 @@
 $system__database_upgrades=array();
 include ("../config/dbupdates.php");
 
```

Comment:
There will need to be some database updates to make sure that the feature is fully available to users who have a running ORSEE installation/database.